src/components/common/VerifiedIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { Circle, Container, Icon, SquareProps, Tooltip } from "@chakra-ui/react"
+import { Check, CircleWavy } from "phosphor-react"
+
+type Props = {
+  iconSize: number
+} & SquareProps
+
+const VerifiedIcon = ({ iconSize, ...chakraProps }: Props): JSX.Element => (
+  <Tooltip label="Verified" hasArrow>
+    <Circle {...chakraProps}>
+      <Container position="relative" h={iconSize} w={iconSize} p={0}>
+        <Icon
+          as={CircleWavy}
+          boxSize={iconSize}
+          color={"blue.500"}
+          weight="fill"
+          position="absolute"
+        />
+        <Icon
+          as={Check}
+          boxSize={iconSize / 2}
+          color={"white"}
+          weight="bold"
+          position="absolute"
```

Comment:
I'd refactor this component a bit. We could use a simple `Flex` or a `Center` component as a wrapper (we don't need the circular shape anymore), then remove the `Container`, and then remove the absolute position from the `Check` icon (it should be centered by default, since it's inside a flexbox)